build.gradle

```diff
@@ -79,6 +79,7 @@ dependencies {
     implementation project(':alert-common')
     implementation project(':alert-database')
     implementation project(':alert-issuetracker')
+    implementation project(':alert-issuetracker-jira')
```

Comment:
is the alert-issuetracker import still necessary if we're pulling in the jira version?

Reply:
yes I should be able just include alert-issuetracker-jira and get the issuetracker project as a transitive.